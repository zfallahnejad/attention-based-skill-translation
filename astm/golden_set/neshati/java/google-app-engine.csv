248432,google-app-engine
12030,google-app-engine
821657,google-app-engine
69258,google-app-engine
635982,google-app-engine
1490882,google-app-engine
119365,google-app-engine
62288,google-app-engine
498860,google-app-engine
571407,google-app-engine
157882,google-app-engine
48387,google-app-engine
130929,google-app-engine
439317,google-app-engine
222467,google-app-engine
14955,google-app-engine
83406,google-app-engine
987490,google-app-engine
1258245,google-app-engine
105744,google-app-engine
656408,google-app-engine
280474,google-app-engine
95810,google-app-engine
867521,google-app-engine
820142,google-app-engine
8418,google-app-engine
23786,google-app-engine
313137,google-app-engine
