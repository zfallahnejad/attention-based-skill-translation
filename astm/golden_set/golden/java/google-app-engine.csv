248432,163,158,0.5077881619937694
12030,134,134,0.5
821657,63,65,0.4921875
69258,57,62,0.4789915966386555
635982,47,29,0.618421052631579
1490882,41,51,0.44565217391304346
119365,36,24,0.6
62288,33,33,0.5
498860,31,11,0.7380952380952381
157882,25,17,0.5952380952380952
48387,25,25,0.5
571407,25,13,0.6578947368421053
439317,23,13,0.6388888888888888
130929,23,23,0.5
2848676,21,8,0.7241379310344828
222467,21,21,0.5
14955,20,14,0.5882352941176471
83406,17,20,0.4594594594594595
987490,16,13,0.5517241379310345
656408,15,13,0.5357142857142857
1258245,15,14,0.5172413793103449
105744,15,9,0.625
95810,14,11,0.56
280474,14,5,0.7368421052631579
867521,14,14,0.5
150016,13,18,0.41935483870967744
820142,13,12,0.52
8418,13,16,0.4482758620689655
23786,11,5,0.6875
313137,11,13,0.4583333333333333
116472,10,6,0.625
1337127,10,14,0.4166666666666667
1165922,10,0,1.0
752918,10,14,0.4166666666666667
5849,10,8,0.5555555555555556
1705598,10,7,0.5882352941176471
164602,10,6,0.625
